src/components/widgets/status/PrintStatus.vue

```diff
@@ -20,7 +20,8 @@
               </template>
               <span>{{ $t('app.printer.status.time_left') }}</span>
             </v-tooltip>
-            {{ timeEstimates.remaining }}
+            <span>{{ timeEstimates.remaining }}</span>
+            <span class="grey--text text--darken-2" v-if="printerPrinting && printTimeEstimationsType !== 'totals'"> / {{ timeEstimates.endTime }}</span>
```

Comment:
It's unclear to me if `printTimeEstimationsType !== 'totals'` is redundant when checking for `printerPrinting`

Reply:
Upon more digging it doesn't seem redundant but I'll leave this comment up for posterity